## `nixpkgs-review` result

Generated using [`nixpkgs-review-gha`](https://github.com/Defelo/nixpkgs-review-gha)

Command: `nixpkgs-review pr 513973`
Commit: [`0fa69e46dcae5fd82c6e989af230d7ccaa732123`](https://github.com/NixOS/nixpkgs/commit/0fa69e46dcae5fd82c6e989af230d7ccaa732123) ([subsequent changes](https://github.com/NixOS/nixpkgs/compare/0fa69e46dcae5fd82c6e989af230d7ccaa732123..pull/513973/head))
Merge: [`3a28f9f34ceff5f1e41130acccbf70620d45ca04`](https://github.com/NixOS/nixpkgs/commit/3a28f9f34ceff5f1e41130acccbf70620d45ca04)

Logs: https://github.com/lonerOrz/nixpkgs-review-gha/actions/runs/24989503625


---
### `x86_64-linux`
<details>
  <summary>:x: 3 packages failed to build:</summary>
  <ul>
    <li>cfclient</li>
    <li>cfclient.dist</li>
    <li>gnuradioPackages.fosphor</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 34 packages built:</summary>
  <ul>
    <li>gnss-sdr</li>
    <li>gnuradio</li>
    <li>gnuradio.man</li>
    <li>gnuradioPackages.bladeRF</li>
    <li>gnuradioPackages.gr-difi</li>
    <li>gnuradioPackages.lora_sdr</li>
    <li>gnuradioPackages.lora_sdr.dev</li>
    <li>gnuradioPackages.osmosdr</li>
    <li>gnuradioPackages.osmosdr.dev</li>
    <li>hifiscan</li>
    <li>hifiscan.dist</li>
    <li>inkcut</li>
    <li>inkcut.dist</li>
    <li>inkscape-extensions.inkcut</li>
    <li>opcua-client-gui</li>
    <li>opcua-client-gui.dist</li>
    <li>python313Packages.enamlx</li>
    <li>python313Packages.enamlx.dist</li>
    <li>python313Packages.pglive</li>
    <li>python313Packages.pglive.dist</li>
    <li>python313Packages.pyqtgraph</li>
    <li>python313Packages.pyqtgraph.dist</li>
    <li>python313Packages.stytra</li>
    <li>python313Packages.stytra.dist</li>
    <li>python314Packages.enamlx</li>
    <li>python314Packages.enamlx.dist</li>
    <li>python314Packages.pglive</li>
    <li>python314Packages.pglive.dist</li>
    <li>python314Packages.pyqtgraph</li>
    <li>python314Packages.pyqtgraph.dist</li>
    <li>python314Packages.stytra</li>
    <li>python314Packages.stytra.dist</li>
    <li>qradiolink</li>
    <li>trunk-recorder</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `x86_64-linux`</summary>
<details>
<summary>gnuradioPackages.fosphor</summary>
<pre>Call Stack (most recent call first):
  /nix/store/drywpnahhww2xyghl861rlkh32arzzsx-gnuradio-3.10.12.0/lib/cmake/gnuradio/GnuradioConfig.cmake:37 (include)
  CMakeLists.txt:34 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found PythonLibs: /nix/store/0r6k8xa2kgqyp3r4v2w7yrb80ma2iawm-python3-3.13.12/lib/libpython3.13.so (found suitable exact version &quot;3.13.12&quot;)
-- Could NOT find Git (missing: GIT_EXECUTABLE) 
CMake Warning (dev) at CMakeLists.txt:94 (find_package):
  Policy CMP0167 is not set: The FindBoost module is removed.  Run &quot;cmake
  --help-policy CMP0167&quot; for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Could NOT find Boost: missing: system (found /nix/store/f1jzlprx190bfspq1i56bd0r8q9lwy0x-boost-1.89.0-dev/lib/cmake/Boost-1.89.0/BoostConfig.cmake (found suitable version &quot;1.89.0&quot;, minimum required is &quot;1.65&quot;))
CMake Error at CMakeLists.txt:97 (message):
  Boost required to compile gr-fosphor


-- Configuring incomplete, errors occurred!</pre>
</details>
</details>

---
### `aarch64-linux`
<details>
  <summary>:x: 3 packages failed to build:</summary>
  <ul>
    <li>cfclient</li>
    <li>cfclient.dist</li>
    <li>gnuradioPackages.fosphor</li>
  </ul>
</details>
<details>
  <summary>:white_check_mark: 34 packages built:</summary>
  <ul>
    <li>gnss-sdr</li>
    <li>gnuradio</li>
    <li>gnuradio.man</li>
    <li>gnuradioPackages.bladeRF</li>
    <li>gnuradioPackages.gr-difi</li>
    <li>gnuradioPackages.lora_sdr</li>
    <li>gnuradioPackages.lora_sdr.dev</li>
    <li>gnuradioPackages.osmosdr</li>
    <li>gnuradioPackages.osmosdr.dev</li>
    <li>hifiscan</li>
    <li>hifiscan.dist</li>
    <li>inkcut</li>
    <li>inkcut.dist</li>
    <li>inkscape-extensions.inkcut</li>
    <li>opcua-client-gui</li>
    <li>opcua-client-gui.dist</li>
    <li>python313Packages.enamlx</li>
    <li>python313Packages.enamlx.dist</li>
    <li>python313Packages.pglive</li>
    <li>python313Packages.pglive.dist</li>
    <li>python313Packages.pyqtgraph</li>
    <li>python313Packages.pyqtgraph.dist</li>
    <li>python313Packages.stytra</li>
    <li>python313Packages.stytra.dist</li>
    <li>python314Packages.enamlx</li>
    <li>python314Packages.enamlx.dist</li>
    <li>python314Packages.pglive</li>
    <li>python314Packages.pglive.dist</li>
    <li>python314Packages.pyqtgraph</li>
    <li>python314Packages.pyqtgraph.dist</li>
    <li>python314Packages.stytra</li>
    <li>python314Packages.stytra.dist</li>
    <li>qradiolink</li>
    <li>trunk-recorder</li>
  </ul>
</details>

---
<details>
<summary>Error logs: `aarch64-linux`</summary>
<details>
<summary>gnuradioPackages.fosphor</summary>
<pre>Call Stack (most recent call first):
  /nix/store/xcsvhnnqh96j0drdmk4g450hx26nnm22-gnuradio-3.10.12.0/lib/cmake/gnuradio/GnuradioConfig.cmake:37 (include)
  CMakeLists.txt:34 (find_package)
This warning is for project developers.  Use -Wno-dev to suppress it.

-- Found PythonLibs: /nix/store/36mns0lq3c2zgky08rsq0xvamfzglzvj-python3-3.13.12/lib/libpython3.13.so (found suitable exact version &quot;3.13.12&quot;)
-- Could NOT find Git (missing: GIT_EXECUTABLE) 
CMake Warning (dev) at CMakeLists.txt:94 (find_package):
  Policy CMP0167 is not set: The FindBoost module is removed.  Run &quot;cmake
  --help-policy CMP0167&quot; for policy details.  Use the cmake_policy command to
  set the policy and suppress this warning.

This warning is for project developers.  Use -Wno-dev to suppress it.

-- Could NOT find Boost: missing: system (found /nix/store/z0lhkr2ald5wrz5ziffv1glvc3n0qcim-boost-1.89.0-dev/lib/cmake/Boost-1.89.0/BoostConfig.cmake (found suitable version &quot;1.89.0&quot;, minimum required is &quot;1.65&quot;))
CMake Error at CMakeLists.txt:97 (message):
  Boost required to compile gr-fosphor


-- Configuring incomplete, errors occurred!</pre>
</details>
</details>

---
### `x86_64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 24 packages failed to build:</summary>
  <ul>
    <li>gnuradio</li>
    <li>gnuradio.man</li>
    <li>gnuradioPackages.lora_sdr</li>
    <li>gnuradioPackages.lora_sdr.dev</li>
    <li>gnuradioPackages.osmosdr</li>
    <li>gnuradioPackages.osmosdr.dev</li>
    <li>hifiscan</li>
    <li>hifiscan.dist</li>
    <li>inkcut</li>
    <li>inkcut.dist</li>
    <li>inkscape-extensions.inkcut</li>
    <li>python313Packages.enamlx</li>
    <li>python313Packages.enamlx.dist</li>
    <li>python313Packages.pglive</li>
    <li>python313Packages.pglive.dist</li>
    <li>python313Packages.pyqtgraph</li>
    <li>python313Packages.pyqtgraph.dist</li>
    <li>python314Packages.enamlx</li>
    <li>python314Packages.enamlx.dist</li>
    <li>python314Packages.pglive</li>
    <li>python314Packages.pglive.dist</li>
    <li>python314Packages.pyqtgraph</li>
    <li>python314Packages.pyqtgraph.dist</li>
    <li>trunk-recorder</li>
  </ul>
</details>

---
### `aarch64-darwin` (sandbox = relaxed)
<details>
  <summary>:x: 28 packages failed to build:</summary>
  <ul>
    <li>gnuradio</li>
    <li>gnuradio.man</li>
    <li>gnuradioPackages.lora_sdr</li>
    <li>gnuradioPackages.lora_sdr.dev</li>
    <li>gnuradioPackages.osmosdr</li>
    <li>gnuradioPackages.osmosdr.dev</li>
    <li>hifiscan</li>
    <li>hifiscan.dist</li>
    <li>inkcut</li>
    <li>inkcut.dist</li>
    <li>inkscape-extensions.inkcut</li>
    <li>python313Packages.enamlx</li>
    <li>python313Packages.enamlx.dist</li>
    <li>python313Packages.pglive</li>
    <li>python313Packages.pglive.dist</li>
    <li>python313Packages.pyqtgraph</li>
    <li>python313Packages.pyqtgraph.dist</li>
    <li>python313Packages.stytra</li>
    <li>python313Packages.stytra.dist</li>
    <li>python314Packages.enamlx</li>
    <li>python314Packages.enamlx.dist</li>
    <li>python314Packages.pglive</li>
    <li>python314Packages.pglive.dist</li>
    <li>python314Packages.pyqtgraph</li>
    <li>python314Packages.pyqtgraph.dist</li>
    <li>python314Packages.stytra</li>
    <li>python314Packages.stytra.dist</li>
    <li>trunk-recorder</li>
  </ul>
</details>

--- a/pkgs/development/python-modules/pyqtgraph/default.nix
+++ b/pkgs/development/python-modules/pyqtgraph/default.nix
@@ -70,10 +70,9 @@ buildPythonPackage (finalAttrs: {
   ];
 
   disabledTests = [
-    # ZeroDivisionError: float division by zero
+    # See https://github.com/pyqtgraph/pyqtgraph/issues/3485
     "test_maps_tick_values_to_local_times"
     "test_maps_hour_ticks_to_local_times_when_skip_greater_than_one"
-
   ]
   ++ lib.optionals (!stdenv.hostPlatform.isx86) [
     # small precision-related differences on other architectures,

--- a/pkgs/development/python-modules/pyqtgraph/default.nix
+++ b/pkgs/development/python-modules/pyqtgraph/default.nix
@@ -26,7 +26,7 @@
 let
   fontsConf = makeFontsConf { fontDirectories = [ freefont_ttf ]; };
 in
-buildPythonPackage rec {
+buildPythonPackage (finalAttrs: {
   pname = "pyqtgraph";
   version = "0.14.0";
   pyproject = true;
@@ -34,7 +34,7 @@ buildPythonPackage rec {
   src = fetchFromGitHub {
     owner = "pyqtgraph";
     repo = "pyqtgraph";
-    tag = "pyqtgraph-${version}";
+    tag = "pyqtgraph-${finalAttrs.version}";
     hash = "sha256-T5rhaBtcKP/sYjCmYNMYR0BGttkiLhWTfEbZNeAdJJ0=";
   };
 
@@ -89,7 +89,7 @@ buildPythonPackage rec {
   meta = {
     description = "Scientific Graphics and GUI Library for Python";
     homepage = "https://www.pyqtgraph.org/";
-    changelog = "https://github.com/pyqtgraph/pyqtgraph/blob/${src.tag}/CHANGELOG";
+    changelog = "https://github.com/pyqtgraph/pyqtgraph/blob/${finalAttrs.src.tag}/CHANGELOG";
     license = lib.licenses.mit;
     platforms = lib.platforms.unix;
     maintainers = with lib.maintainers; [
@@ -97,4 +97,4 @@ buildPythonPackage rec {
       doronbehar
     ];
   };
-}
+})

--- a/pkgs/development/python-modules/pyqtgraph/default.nix
+++ b/pkgs/development/python-modules/pyqtgraph/default.nix
@@ -73,6 +73,8 @@ buildPythonPackage (finalAttrs: {
     # See https://github.com/pyqtgraph/pyqtgraph/issues/3485
     "test_maps_tick_values_to_local_times"
     "test_maps_hour_ticks_to_local_times_when_skip_greater_than_one"
+    "test_plotscene"
+    "test_simple"
   ]
   ++ lib.optionals (!stdenv.hostPlatform.isx86) [
     # small precision-related differences on other architectures,

